### ISSUE TITLE
Add test for 0-RTT support in TLS 1.3 servers

### ISSUE THREAD

**Issue by shimile** [ssllabs/ssllabs-scan#683]
I think that due to replay attacks nature of 0-RTT, perhaps also make it as 2 distinct tests: "0-RTT Support only in GET requests" to "0-RTT Support in NON-GET requests" (which IMHO should be a warning?)

**Comment by tamthing** [comment]
We have a work in progress for 0-RTT detection. We would further update once it is released.

**Comment by felixbuenemann** [comment]
The 0-RTT test currently fails with nginx 1.15.9 / openssl 1.1.1 with both `TLSv1.3` in `ssl_protocols` and `ssl_early_data` enabled:

https://dev.ssllabs.com/ssltest/analyze.html?d=ssltest.louis.info&hideResults=on

The Protocol Details section lists: `0-RTT enabled: No`

However the following test seems to indicate it is working:

```bash
host=ssltest.louis.info
echo -e "GET / HTTP/1.1\r\nHost: $host\r\nConnection: close\r\n\r\n" > request.txt
openssl s_client -connect $host:443 -tls1_3 -sess_out session.pem -ign_eof < request.txt
openssl s_client -connect $host:443 -tls1_3 -sess_in session.pem -early_data request.txt
```

The server accepts the request send in the early data:

```
Reused, TLSv1.3, Cipher is TLS_AES_256_GCM_SHA384
Server public key is 256 bit
Secure Renegotiation IS NOT supported
Compression: NONE
Expansion: NONE
No ALPN negotiated
Early data was accepted
Verify return code: 0 (ok)
---
HTTP/1.1 200 OK
Server: nginx/1.15.9
```

**Comment by vyasatavi** [comment]
SSLLabs report now shows Yes for early_data. Was there a fix done for it?

**Comment by tamthing** [comment]
@vyasatavi  Yes, it has been included since version 1.33.1. https://community.qualys.com/docs/DOC-5737-ssl-labs-changelog
